resources/Microsoft.Windows.Setting.Accessibility/Microsoft.Windows.Setting.Accessibility.psm1

```diff
@@ -241,6 +250,57 @@ class MousePointer {
     }
 }
 
+[DSCResource()]
+class Scrollbars {
+
+    [DscProperty(Key)] [Status] $AlwaysShow = [Status]::KeepCurrentValue

```

Comment:
```suggestion
    [DscProperty(Key)] [Bool] $Show = $false

```

Since there are only two states we can just use true or false.

Reply:
The same argument can be made for any of the other enums in this file. I think it's better to retain `KeepCurrentValue`, both to retain the functionality and to remain consistency with the other Resources in this file.